src/cljs/rems/spa.cljs

```diff
@@ -255,6 +254,8 @@
     (events/listen
      HistoryEventType/NAVIGATE
      (fn [event]
+       (when (and js/window.rems js/window.rems.hooks js/window.rems.hooks.navigate)
+         (js/window.rems.hooks.navigate (.-token event)))
```

Comment:
The conditionals here and with other hooks could be avoided by always having hooks; the default hooks would be just no-ops.